mava/utils/jax_training_utils.py

```diff
@@ -46,42 +49,88 @@ def compute_running_mean_var_count(
     M2 = m_a + m_b + jnp.square(delta) * count * batch_count / tot_count
     new_var = M2 / tot_count
     new_count = tot_count
+    new_std = jnp.sqrt(new_var)
 
-    return jnp.array([new_mean, new_var, new_count])
+    return dict(mean=new_mean, var=new_var, std=new_std, count=new_count)
 
 
-def normalize(stats: jnp.ndarray, batch: jnp.ndarray) -> jnp.ndarray:
+def normalize(stats: Any, batch: jnp.ndarray) -> jnp.ndarray:
```

Comment:
change the typing to `Dict[str, float]` unless it can be anything else?

Although if this was a `jnp.array` we could jit this function :wink:

Reply:
So the stats will be arrays for observations and single numbers for target values.

Reply:
Ok then can you do `Dict[str, Union[jnp.array, float]]`?

Reply:
Okay I will do that.